etcdutl/snapshot/v3_snapshot.go

```diff
@@ -156,8 +158,17 @@ func (s *v3Manager) Status(dbPath string) (ds Status, err error) {
 					return fmt.Errorf("cannot write to bucket %s", herr.Error())
 				}
 				if iskeyb {
-					rev := mvcc.BytesToRev(k)
+					var rev mvcc.BucketKey
+					rev, err = mvcc.BytesToBucketKey(k)
+					if err != nil {
+						return fmt.Errorf("cannot parse revision key : %s", err.Error())
```

Comment:
can you %w the error please? and I assume you wanted to format the bucket key (k) as the %s?

Reply:
Done.